From edb431ebc8be9f2ab47c2ea5e585a6ba8ac3a16e Mon Sep 17 00:00:00 2001
From: Andy Green <andy@openmoko.com>
Date: Sun, 10 Aug 2008 09:14:15 +0100
Subject: [PATCH] checkpatch-fixes.patch

This cleans out some random externs in C files that checkpatch does not like
and introduces a couple of .h files to contain them.  Plus some other minor
checkpatch style complaints.

Signed-off-by: Andy Green <andy@openmoko.com>
---
 arch/arm/mach-s3c2410/mach-gta01.c     |    4 ++--
 arch/arm/mach-s3c2440/mach-gta02.c     |    4 ++--
 arch/arm/plat-s3c24xx/neo1973_pm_gsm.c |    6 +-----
 drivers/video/backlight/gta01_bl.c     |    2 +-
 4 files changed, 6 insertions(+), 10 deletions(-)

diff --git a/arch/arm/mach-s3c2410/mach-gta01.c b/arch/arm/mach-s3c2410/mach-gta01.c
index 4927cd4..a364b76 100644
--- a/arch/arm/mach-s3c2410/mach-gta01.c
+++ b/arch/arm/mach-s3c2410/mach-gta01.c
@@ -73,6 +73,8 @@
 #include <asm/plat-s3c24xx/pm.h>
 #include <asm/plat-s3c24xx/udc.h>
 #include <asm/plat-s3c24xx/neo1973.h>
+#include <asm/arch-s3c2410/neo1973-pm-gsm.h>
+
 #include <linux/jbt6k74.h>
 
 static struct map_desc gta01_iodesc[] __initdata = {
@@ -666,8 +668,6 @@ static void __init gta01_map_io(void)
 	s3c24xx_init_uarts(gta01_uartcfgs, ARRAY_SIZE(gta01_uartcfgs));
 }
 
-extern int gta_gsm_interrupts;
-
 static irqreturn_t gta01_modem_irq(int irq, void *param)
 {
 	printk(KERN_DEBUG "GSM wakeup interrupt (IRQ %d)\n", irq);
diff --git a/arch/arm/mach-s3c2440/mach-gta02.c b/arch/arm/mach-s3c2440/mach-gta02.c
index b37d1c4..1658fa0 100644
--- a/arch/arm/mach-s3c2440/mach-gta02.c
+++ b/arch/arm/mach-s3c2440/mach-gta02.c
@@ -78,6 +78,8 @@
 #include <asm/plat-s3c24xx/pm.h>
 #include <asm/plat-s3c24xx/udc.h>
 #include <asm/plat-s3c24xx/neo1973.h>
+#include <asm/arch-s3c2410/neo1973-pm-gsm.h>
+
 #include <linux/jbt6k74.h>
 
 #include <linux/glamofb.h>
@@ -1531,8 +1533,6 @@ static void __init gta02_map_io(void)
 	s3c24xx_init_uarts(gta02_uartcfgs, ARRAY_SIZE(gta02_uartcfgs));
 }
 
-extern int gta_gsm_interrupts;
-
 static irqreturn_t gta02_modem_irq(int irq, void *param)
 {
 	printk(KERN_DEBUG "modem wakeup interrupt\n");
diff --git a/arch/arm/plat-s3c24xx/neo1973_pm_gsm.c b/arch/arm/plat-s3c24xx/neo1973_pm_gsm.c
index 8d5e41c..9873568 100644
--- a/arch/arm/plat-s3c24xx/neo1973_pm_gsm.c
+++ b/arch/arm/plat-s3c24xx/neo1973_pm_gsm.c
@@ -18,12 +18,12 @@
 #include <linux/console.h>
 #include <linux/errno.h>
 #include <linux/interrupt.h>
-#include <linux/resume-dependency.h>
 
 #include <asm/gpio.h>
 #include <asm/mach-types.h>
 #include <asm/arch/gta01.h>
 #include <asm/plat-s3c24xx/neo1973.h>
+#include <asm/arch/s3c24xx-serial.h>
 
 #ifdef CONFIG_MACH_NEO1973_GTA02
 #include <asm/arch/gta02.h>
@@ -31,10 +31,6 @@
 #include <asm/arch/regs-gpioj.h>
 #endif
 
-extern void s3c24xx_serial_console_set_silence(int silence);
-extern void s3c24xx_serial_register_resume_dependency(struct resume_dependency *
-					     resume_dependency, int uart_index);
-
 int gta_gsm_interrupts;
 EXPORT_SYMBOL(gta_gsm_interrupts);
 
diff --git a/drivers/video/backlight/gta01_bl.c b/drivers/video/backlight/gta01_bl.c
index 34c19c8..a7170df 100644
--- a/drivers/video/backlight/gta01_bl.c
+++ b/drivers/video/backlight/gta01_bl.c
@@ -152,7 +152,7 @@ EXPORT_SYMBOL_GPL(gta01bl_deferred_resume);
 
 static int gta01bl_resume(struct platform_device *dev)
 {
-	if (! gta01bl_defer_resume_backlight)
+	if (!gta01bl_defer_resume_backlight)
 		gta01bl_deferred_resume();
 	return 0;
 }
-- 
1.5.6.3

